ext/openssl/openssl.c

```diff
@@ -5142,6 +5275,34 @@ PHP_FUNCTION(openssl_get_cipher_methods)
 }
 /* }}} */
 
+/* {{{ proto array openssl_get_curve_names([bool aliases = false])
+   Return array of available elliptic curves */
+#ifdef HAVE_EVP_PKEY_EC
+PHP_FUNCTION(openssl_get_curve_names)
+{
+	EC_builtin_curve *curves = NULL;
+	const char *sname;
+	int i;
+
+	array_init(return_value);
+	size_t len = EC_get_builtin_curves(NULL, 0);
+	curves = emalloc((int)(sizeof(EC_builtin_curve) * len));
+	if (!EC_get_builtin_curves(curves, len)) {
+		return;
+	}
+
+	for(i = 0; i < len; i++)
+	{
+		sname = OBJ_nid2sn(curves[i].nid);
+		if (sname == NULL)
```

Comment:
missing braces (this is required in our coding style even if it's not needed here by C standard...)